eos_data_distribution/producers/usb.py

```diff
@@ -107,7 +107,6 @@ def main():
     GLib.unix_signal_add(GLib.PRIORITY_DEFAULT, signal.SIGTERM, signal_cb)
     monitor = Gio.VolumeMonitor.get()
     store = SimpleStoreProducer(prefix=SUBSCRIPTIONS_SOMA,
-                                split=defaults.ENDLESS_NDN_CACHE_PATH,
```

Comment:
you need split !!!
this is what maps `/media/blah/usb/some/dir/soma/ndn/files` to -> `/soma/ndn/files`
this looks very wrong.